probe/probe.go

```diff
@@ -81,23 +83,40 @@ func PrintProbeResult(c *k8s.Client, o Options) error {
 		}
 	}
 	if isSystemdMode() {
-		err := probeSystemdMode()
+		err := probeSystemdMode(o)
 		if err != nil {
 			return err
 		}
 		return nil
 	}
-	if isKubeArmorRunning(c, o) {
-		getKubeArmorDeployments(c, o)
-		getKubeArmorContainers(c, o)
-		err := probeRunningKubeArmorNodes(c, o)
+	isRunning, daemonsetStatus := isKubeArmorRunning(c, o)
+	if isRunning {
```

Comment:
shouldn't we check the opposite, log the error, and return if kubearmor is not running?

Reply:
from earlier code structure, if kubearmor is not running, we still are executing the `if..else`  statements below 🤔